app/src/main/java/org/fossasia/pslab/fragment/InstrumentsFragment.java

```diff
@@ -80,6 +81,10 @@ public void onItemClick(ApplicationItem item) {
                                 intent = new Intent(context, PowerSourceActivity.class);
                                 startActivity(intent);
                                 break;
+                            case "Lux Meter":
+                                intent = new Intent(context, LuxMeterActivity.class);
+                                startActivity(intent);
+                                break;
                         }
 
```

Comment:
Extra lines not needed

Reply:
This was not done by me as you can see there are no edit lines. I would like to keep it that way so there won't be merge conflicts with others.

Reply:
Guys we need to review a little more on actual code. Spacing and refactoring is OK to review but please focus more on `Java`